opentracing-jaxrs2/src/main/java/io/opentracing/contrib/jaxrs2/server/ServerTracingFilter.java

```diff
@@ -59,7 +57,8 @@ public void filter(ContainerRequestContext requestContext) throws IOException {
                 spanBuilder.asChildOf(extractedSpanContext);
             }
 
-            Span span = spanBuilder.start();
+            Span span = spanBuilder.startManual();
+            tracer.makeActive(span);
```

Comment:
What do you mean e.g. could you outline possible solution? What should be done in request and response filters.